package-lock.json

```diff
@@ -12919,4 +12919,4 @@
       }
     }
   }
-}
+}
```

Comment:
seems like you removed new line thats why this file is still showing in diff. can you fix this ?